ecsclient/common/exceptions.py

```diff
@@ -70,36 +70,34 @@ def from_response(cls, resp, message=None):
                    http_response_headers=resp.headers)
 
     def __str__(self):
-        a = self.message
+        a = "\n\nError Message: %s\n" % self.message
         if self.ecs_details not in a:
-            a += ' - %s' % self.ecs_details
+            a += 'ECS_details: %s\n' % self.ecs_details
         if self.ecs_description not in a:
-            a += ' - %s' % self.ecs_description
+            a += 'ECS_description: %s\n\n' % self.ecs_description
         b = ''
         if self.http_scheme:
-            b += '%s://' % self.http_scheme
+            b += 'HTTP/S Request/Response Details:\n---------------------------------\nScheme: %s\n' % self.http_scheme
         if self.http_host:
-            b += self.http_host
+            b += "Host: %s\n" % self.http_host
         if self.http_port:
-            b += ':%s' % self.http_port
+            b += 'Port:%s\n' % self.http_port
         if self.http_path:
-            b += self.http_path
+            b += 'Path: %s\n' % self.http_path
         if self.http_query:
-            b += '?%s' % self.http_query
+            b += 'Query: %s\n' % self.http_query
+
         if self.http_status:
             if b:
-                b = '%s %s' % (b, self.http_status)
+                b += 'Response_Status: %s\n' % str(self.http_status)
             else:
-                b = str(self.http_status)
+                b = 'HTTP/S Response_Status: %s\n' % str(self.http_status)
         if self.http_reason:
             if b:
-                b = '%s %s' % (b, self.http_reason)
+                b += 'Response_Reason: %s\n' % self.http_reason
             else:
-                b = '- %s' % self.http_reason
+                b = 'HTTP/S Response_Reason: %s\n' % self.http_reason
         if self.http_response_content:
-            if len(self.http_response_content) <= 60:
-                b += '   %s' % self.http_response_content
-            else:
-                b += '  [first 60 chars of response] %s' \
-                     % self.http_response_content[:60]
-        return b and '%s: %s' % (a, b) or a
+            b += 'Response_content: %s' % self.http_response_content
```

Comment:
Recommend re-adding size conditional, but increasing size to something larger, such as 8k.

Reply:
Added 8K limit
https://github.com/EMCECS/python-ecsclient/pull/63/commits/a36e31af0fb5d07055bfa67519e1a83652c2771d